JudoPayDotNet/Models/CompleteThreeDSecureModel.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Runtime.Serialization;
+
+namespace JudoPayDotNet.Models
+{
+    /// <summary>
+    /// Fields to complete a ThreeDSecure Two transaction
+    /// </summary>
+    [DataContract(Name = "CompleteThreeDSecureModel", Namespace = "")]
+    public class CompleteThreeDSecureModel
```

Comment:
Maybe this should be called CompleteThreeDSecureTwoModel just to be explicit about it's purpose